src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -54,5 +54,9 @@ trait MicrositeKeys {
   val micrositePalette     = settingKey[Map[String, String]]("Microsite palette")
   val micrositeGithubOwner = settingKey[String]("Microsite Github owner")
   val micrositeGithubRepo  = settingKey[String]("Microsite Github repo")
+  val micrositeGitHostingService =
+    settingKey[String]("Service used for git hosting. By default, it'll be 'Github'.")
+  val micrositeGitHostingUrl = settingKey[String](
```

Comment:
can we set this automatically to empty strings when the user specifies a different `micrositeGitHostingService` automatically so they don't have to do that themselves?